backend/app/assets/stylesheets/spree/backend/shared/_tables.scss

```diff
@@ -1,14 +1,20 @@
 table {
-  width: 100%;
-  margin-bottom: 15px;
-  border-collapse: separate;
+  // Extend all tables with Bootstrap's default table style
+  @extend .table;
```

Comment:
This isn't worse than what was here before (bootstrap's `.table` sets pretty much the same CSS as we already had here). It would be ideal if we were to move away from styling all `table` elements like this and instead required a class. That way we avoid needing to un-style the styles when we want a table to be "normal".